site/lists/jobs.scroll

```diff
@@ -6,30 +6,38 @@ columnWidth 60
 
 pipeTable
  site|siteLink
+ Arc|https://arc.dev/
+ Berlinstartupjobs|https://berlinstartupjobs.com/
+ Braintrust|https://www.usebraintrust.com/
  Craigslist|https://www.craigslist.org/
  Dice|https://www.dice.com/
+ Dribbble|https://dribbble.com/jobs?location=Anywhere
+ Flexjobs|https://www.flexjobs.com/ 
  GitHub Jobs|https://jobs.github.com/
  Glassdoor|https://www.glassdoor.com/
  Hacker News Jobs|https://news.ycombinator.com/jobs
+ Hubstaff|https://talent.hubstaff.com/
  Indeed|http://indeed.com/
+ Jobspresso|https://jobspresso.co/
  LinkedIn|https://www.linkedin.com/
+ Monster|https://www.monster.com/
```

Comment:
of course! we should probably look and see if they have API integrations we could use

Reply:
Yes, Monster is the golden-oldie of online job sites!

Reply:
okay, let me check if they have APIs